libraries/AP_DDS/gen_config_h.py

```diff
@@ -31,17 +31,18 @@
 }
 
 defines = {
-    "UCLIENT_PROFILE_UDP": 0,
+    "UCLIENT_PROFILE_UDP": 1,
```

Comment:
Can we drive this setting from whether UDP is enabled in the build? Otherwise, I think there will be overhead for all users, whether they use UDP or not.

Reply:
I checked and it has zero flash cost to have this set to 1 and not use it

Reply:
Excellent.